stories/2-Card.stories.js

```diff
@@ -279,3 +280,25 @@ export const IssuesCard = () => (
     </IssuesColumn>
   </Background>
 );
+
+export const OnBoardCard = () => (
+  <Background style={{height: 1024}}>
+    <OnBoardStyle>
+      <h4>Onboarding</h4>
+      <div className="checkbox">
+        <input id="agenda" type="checkbox" />
+        <label htmlFor="agenda">Follow the babel community</label>
```

Comment:
👍 for leveraging semantic html. This is need for a11y. I did not require it for this project since I only just figured it out, but this will pass [my a11y tests](https://github.com/bdougie/open-sauced/blob/master/src/tests/Footer.test.js#L5-L14) when implemented in the future. 

```
import {axe, toHaveNoViolations} from "jest-axe";
expect.extend(toHaveNoViolations);

test("container component should have no violations", async () => {
  const {container} = render(<OnBoardCard />);
  const results = await axe(container);
  expect(results).toHaveNoViolations();

  cleanup();
});
```